metadata/static/spark/overview.md

```diff
@@ -0,0 +1,26 @@
+## Overview
+Apache Spark is a fast and general-purpose cluster computing system for big data. 
+It provides high-level APIs in Scala, Java, Python, and R, and an optimized engine that supports general computation graphs for data analysis. 
+It also supports a rich set of higher-level tools including: Spark SQL for SQL and DataFrames, MLlib for machine learning, 
+GraphX for graph processing, and Spark Streaming for stream processing.
+
+The KUDO Spark Operator creates, configures, and manages instances of Spark Operator running on Kubernetes.
+
+- Easily create and manage Spark workloads on Kubernetes using `SparkApplication` resource
+- Spark History Server support
+- Metrics collection into Prometheus via ServiceMonitors  
+- Monitor Spark jobs using pre-configured Grafana dashboards
+- High Availability deployment mode
+- Integration with external batch scheduler (Volcano)
+- Graceful rolling updates for any cluster configuration changes
```

Comment:
I'm not sure the "rolling upgrade" is applicable to Spark and suggest removing this bullet

Reply:
oh, right, I believe this is about this: https://kudo.dev/docs/developing-operators/plans.html#parameter-triggers, and we don't use this in Spark operator atm